cumulus_library/studies/core/builder_condition_codableconcept.py

```diff
@@ -0,0 +1,31 @@
+""" Module for generating condition codeableConcept table"""
+
+from cumulus_library.base_table_builder import BaseTableBuilder
+from cumulus_library.template_sql.templates import (
+    CodeableConceptConfig,
+    get_codeable_concept_denormalize_query,
+)
+
+
+class ConditionCodableConceptBuilder(BaseTableBuilder):
+    display_text = "Creating condition code table..."
+
+    def prepare_queries(self, cursor: object, schema: str):
+        """Constructs queries related to condition codeableConcept
+
+        :param cursor: A database cursor object
+        :param schema: the schema/db name, matching the cursor
+
+        """
+        config = CodeableConceptConfig(
+            source_table="condition",
+            source_id="id",
+            cc_columns=[{"name": "code", "is_array": False}],
```

Comment:
these are the only two lines changed in this file, which is renamed from builder_condition_codes.py. not sure why git isn't properly detecting this one.